docs/registry.md

```diff
@@ -52,10 +52,11 @@ KubeKey now allows users to create a container image registry ([Docker registry]
        ## `harbor` can be also set for type.
        # type: "harbor"  
        privateRegistry: dockerhub.kubekey.local
+       privateRegistryIp: ""
        auths:
          "dockerhub.kubekey.local":
-           username: admin
-           password: Harbor12345
+            username: admin
```

Comment:
The same as above.

Reply:
yaml unit 2 space align

Reply:
`runtime.RawExtension` is used here. And both of these cases can be parsed successfully.

https://github.com/kubernetes/apimachinery/blob/v0.23.5/pkg/runtime/types.go#L94

Reply:
Just about code style, Compliant with yaml standard，will be back